pkg/lustre-driver/service/service.go

```diff
@@ -31,16 +31,21 @@ const (
 	VendorVersion = "v0.0.1"
 )
 
-// Service is a CSI SP and idempotency.Provider.
+// Service is a CSI SP and idempotency.Provider
+// Contains 3 interface fields, each of which have a set of functions that must be implemented.
 type Service interface {
 	csi.ControllerServer
 	csi.IdentityServer
 	csi.NodeServer
 }
 
-type service struct{}
+// LustreService implements the Service interface.
```

Comment:
Renamed and capitalized `service` allowing it to be used outside the package. `service` as a struct type conflicts with `service` as a package name, so used `LustreService` instead.

Reply:
Again, not sure why this needs to be exported. If you have a conflicting `service` package you can rename the import to avoid the conflict like `import srvc github.com/service`